site/en/docs/semantic_retriever.ipynb

```diff
@@ -0,0 +1,1031 @@
+{
```

Comment:
<p>Are we able to strip these too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>